docs/source/guide/index.rst

```diff
@@ -380,6 +380,342 @@ Using another computer with access to ``repository-service-tuf-api``
   3.  Run ``rstuf admin ceremony -b -u [-f filename]``
 
 
+.. rstuf-cli-admin-metadata
+
+Metadata Management (``metadata``)
+----------------------------------
+
+.. code::
+
+    ❯ rstuf admin metadata
+
+    Usage: rstuf admin metadata [OPTIONS] COMMAND [ARGS]...
+
+    Token Management.
+
+    ╭─ Options ──────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │  --help  -h    Show this message and exit.                                                                             │
+    ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+    ╭─ Commands ─────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │  update  Start a new metadata update ceremony.                                                                         │
+    ╰────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+
+.. rstuf-cli-admin-metadata-update
+
+update
+......
+
+The metadata update ceremony allows:
+- extending Root expiration
+- modification of the keys used for signing (no matter if Root keys or the
+Online key),
+- changing of the Root threshold value
+
+.. code::
+
+    ❯ rstuf admin metadata update --help
+
+    Usage: rstuf admin metadata update [OPTIONS]
+
+    Start a new metadata update ceremony.
+
+    ╭─ Options ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │ --current-root-uri      TEXT  URL or local path to the current root.json file.                                                                     │
+    │ --file              -f  TEXT  Generate specific JSON payload file [default: metadata-update-payload.json]                                          │
+    │ --upload            -u        Upload existent payload 'file'. Optional '-f/--file' to use non default file name.                                   │
+    │ --run-ceremony                When '--upload' is set this flag can be used to run the ceremony and the result will be uploaded.                    │
+    │ --save              -s        Save a copy of the metadata locally. This option saves the JSON metadata update payload file in the in the current   │
+    │                               directory.                                                                                                           │
+    │ --help              -h        Show this message and exit.                                                                                          │
```

Comment:
First of all, I agree entirely with the suggestion.

However, in this PR, we are not adding or removing those parameters for the `rstuf admin ceremony`. I think it is an update for the documentation only. We should file an issue and add it as an improvement for the `rstuf admin ceremony` parameters, and we can decide if we want these changes on MWV (beta) or later.

The changes for the `rstuf admin metadata upload` could also be a new issue, a small new feature, to avoid extending (make even bigger this PR). If it is a decision to add to this PR, I'm ok as well, but doing it in a specific PR could be faster.

Reply:
As @kairoaraujo has suggested, let's keep the current state as is given that it works and is tested.
I filed an issue about that: https://github.com/repository-service-tuf/repository-service-tuf-cli/issues/298 and we can think about how to solve this problem for both ceremonies.